tests/test_services.py

```
@@ -214,71 +448,52 @@ async def test_service_restart_only_if_running_and_reload_interface(
+        ("_service_start_service", "start_service", "ok-first"),
+        ("_service_start_service", "start_service", "bad-first"),
+        ("_service_stop_service", "stop_service", "ok-first"),
+        ("_service_stop_service", "stop_service", "bad-first"),
+        ("_service_restart_service", "restart_service", "ok-first"),
+        ("_service_restart_service", "restart_service", "bad-first"),
     ],
 )
 async def test_service_start_stop_restart_failure_variants(
-    monkeypatch: pytest.MonkeyPatch, ph_hass: Any, method_name: Any, method_attr: Any
+    monkeypatch: pytest.MonkeyPatch,
+    ph_hass: Any,
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `ph_hass`

(ANN401)